python_orocos_kdl_vendor/package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<?xml-model
+  href="http://download.ros.org/schema/package_format2.xsd"
+  schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>python_orocos_kdl_vendor</name>
+  <version>0.1.0</version>
+  <description>
+    Wrapper around PyKDL, providing nothing but a dependency on PyKDL on some systems.
+    On others, it provides an ExternalProject build of python_orocos_kdl.
```

Comment:
This isn't strictly true anymore, since we are using `FetchContent` rather than `ExternalProject`.  But it is nitpicking.

Reply:
Updated [e21ad5b](https://github.com/ros2/orocos_kdl_vendor/pull/1/commits/e21ad5b2125ae8abcd7a0a96bda9d5ef90901cab)[](https://github.com/jacobperron)